app/uk/gov/hmrc/apidocumentation/controllers/ApiDocumentationController.scala

```diff
@@ -247,7 +228,7 @@ class ApiDocumentationController @Inject() (
       }
 
       val categories = APICategoryFilters.categoryMap.getOrElse(api.name, Seq.empty) ++ api.categories
-      documentationService.fetchApiSpecification(service, version, cacheBuster).flatMap(_.fold(renderOas(categories))(renderRamlSpec))
```

Comment:
Does this mean we no longer have the cacheBuster stuff (tbf it maybe as it looks that Oas never could be cache bust)